### ISSUE TITLE
Gradle - unavailable dependencies when building OpenCms 11

### ISSUE THREAD

**Issue by Morwud** [alkacon/opencms-core#665]
Hello,
I have a problem with building OCMS 11. When I try to build it with
`./gradlew war`
from the branch 'branch_11_0_x' I always get a message about missing some dependencies:
```
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':compileClasspath'.
> Could not resolve org.jpedal.jpedal-lgpl:jpedal-lgpl:4.92b23.
  Required by:
      project :
   > Could not resolve org.jpedal.jpedal-lgpl:jpedal-lgpl:4.92b23.
      > Could not get resource 'http://software.rescarta.org/nexus/content/repositories/thirdparty/org/jpedal/jpedal-lgpl/jpedal-lgpl/4.92b23/jpedal-lgpl-4.92b23.pom'.
         > Could not GET 'http://software.rescarta.org/nexus/content/repositories/thirdparty/org/jpedal/jpedal-lgpl/jpedal-lgpl/4.92b23/jpedal-lgpl-4.92b23.pom'.
            > Connect to software.rescarta.org:80 [software.rescarta.org/192.159.83.79] failed: connect timed out
> Could not resolve org.tepi.filtertable:filteringtable:1.0.1.v8.
  Required by:
      project :
   > Could not resolve org.tepi.filtertable:filteringtable:1.0.1.v8.
      > Could not get resource 'http://software.rescarta.org/nexus/content/repositories/thirdparty/org/tepi/filtertable/filteringtable/1.0.1.v8/filteringtable-1.0.1.v8.pom'.
         > Could not HEAD 'http://software.rescarta.org/nexus/content/repositories/thirdparty/org/tepi/filtertable/filteringtable/1.0.1.v8/filteringtable-1.0.1.v8.pom'.
            > Connect to software.rescarta.org:80 [software.rescarta.org/192.159.83.79] failed: connect timed out
> Could not resolve org.vaadin.addons:popupbutton:3.0.0.
  Required by:
      project :
   > Could not resolve org.vaadin.addons:popupbutton:3.0.0.
      > Could not get resource 'http://software.rescarta.org/nexus/content/repositories/thirdparty/org/vaadin/addons/popupbutton/3.0.0/popupbutton-3.0.0.pom'.
         > Could not HEAD 'http://software.rescarta.org/nexus/content/repositories/thirdparty/org/vaadin/addons/popupbutton/3.0.0/popupbutton-3.0.0.pom'.
            > Connect to software.rescarta.org:80 [software.rescarta.org/192.159.83.79] failed: connect timed out
```
It looks like whatever was located in 'http://software.rescarta.org' is now not available (at least for me).

Is there anything I can do to be able to build the WAR without these dependencies?

**Comment by gWestenberger** [comment]
This seems to have been a temporary error on the Maven repository side. Try the --refresh-dependencies option for gradle.

**Comment by Morwud** [comment]
Tried:
- running with --refresh-dependencies
- clean my local cache with rm -rf $HOME/.gradle/caches/
- clean .m2/.repository folder

Unfortunately nothing helped. I was hoping to get the libs from another resource and add them locally to get the build working but I can't find any other source of the jpedal lib.

**Comment by Yuanuo** [comment]
I had the same problem with jpedal lib.
Then I used this version to solve the problem:（in the **dependencies.gradle** file）

    // distribution group: 'org.jpedal.jpedal-lgpl', name: 'jpedal-lgpl', version: '4.92b23'
    // https://mvnrepository.com/artifact/org.jpedal/jpedal-lgpl
    distribution group: 'org.jpedal', name: 'jpedal-lgpl', version: '4.74b27'

**Comment by tHerrmann** [comment]
You could also use the JAR file from the OpenCms distribution and reference it from the dependencies.gradle file.
Can you access the repository server at all? Is it possible that access is blocked based on your location?

**Comment by Morwud** [comment]
Ok, I finally managed to build OCMS 11.

I used suggestion from @Yuanuo to get older version of the library from other source available to me. Then I commented out the maven repository software.rescarta.org which solved the other two missing dependencies, because they can be found elsewhere (I checked that on mvnrepository.com).

I did forgot about the jar in OpenCms distribution, that could be usable as well. And no, I can't access the repo server at all (although I don't know how to check that other than try the links mentioned above). I tried to access it from 2 different locations.

tl;dr: To fix it I
1. Changed JPedal dependency to older one located on different server
2. Commented out the `software.rescarta.org` repository defined in `build.gradle`

**Comment by trinhbiendich** [comment]
you can use my repo. 
```
jcenter()
    maven {
        url "http://repo.bolacmuito.xyz/release/"
    }
```

**Comment by gWestenberger** [comment]
Fixed in master branch.
